src/game/movement.rs

```diff
@@ -25,9 +32,13 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
+/// This is the input for our character controller,
+/// which in this case is only the direction the character wants to move in.
```

Comment:
```suggestion
/// This component acts as a virtual joystick for the character's movement.
```
Not necessarily this wording, but I want to make it clear that this component doesn't represent _input_, like keyboard / mouse input. It's specifically an intermediate layer that represents how the character is trying to move, and it can be populated in any way: keyboard / mouse input, analog stick input, enemy AI, some special system, etc.

Reply:
[`4771b68` (#230)](https://github.com/TheBevyFlock/bevy_quickstart/pull/230/commits/4771b68b43977c73fedeacb47ac210c3d073effb)